hypervisor/src/kvm/mod.rs

```diff
@@ -314,6 +316,19 @@ impl vm::Vm for KvmVm {
         self.create_device(&mut vfio_dev)
             .map_err(|e| vm::HypervisorVmError::CreatePassthroughDevice(e.into()))
     }
+    ///
+    /// Get the Vm state. Return VM specific data
+    ///
+    fn state(&self) -> vm::Result<VmState> {
+        Ok(self.state)
+    }
+    ///
+    /// Set the VM state
+    ///
+    #[allow(unused_variables)]
+    fn set_state(&self, state: &VmState) -> vm::Result<()> {
```

Comment:
Please remove `#[allow(unused_variables)]` and replace the `state` parameter with `_state`.